ta/os_test/os_test.c

```diff
@@ -80,17 +80,65 @@ while (true) {
 		     nbuf, (unsigned int)res);
 		return res;
 	}
+	if (vblen != strlen(vbuf) + 1) {
+		EMSG("1TEE_GetPropertyAsString(\"%s\") returned length %u instead of %u\n",
```

Comment:
I think it was to better identify the trace. Now EMSG dumps the line, it is useless. I'll remove it.